pkg/emscripten/libretro/indexer.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/node
```

Comment:
``` suggestion
#!/usr/bin/env node
```

Using `env` for the script instead of the bin/node will allow it to work if you're using [nvm](https://github.com/creationix/nvm). See https://x-team.com/blog/a-guide-to-creating-a-nodejs-command/ for more details.